### PR TITLE
GITHUB-334: Update docker-compose YAML to use more recent base-image instead of Alpine

### LINKED ISSUE #334: Update docker-compose-with-jupyter.yml to use base-notebook image
docker-compose-with-jupyter.yml currently uses an alpine Jupyter image and needs to be converted to use the nbgallery/base-notebook image.

### PR DESCRIPTION
**Pull request by nklissus**
This should be a very simple swap and everything seems to work with the base-image just swapped in. This should fix #334

### CODE REVIEW

**Comment by mcrutch** [comment]
@jeffyjefflabs Should we just make the Jupyter container run on 8888 since that is the default anyway and if they look at the logs to get the token, they will just be able to grab that URL rather than have to change the port to 80?

File: docker-compose-with-jupyter.yml [hunk 1]
```diff
@@ -2,13 +2,12 @@ version: "3"
...
 services:
   jupyter:
-    image: nbgallery/jupyter-alpine
+    image: nbgallery/base-notebook
     ports:
-      - "80:80"
+      - "80:8888"
     environment:
       NBGALLERY_URL: http://localhost:3000
       NBGALLERY_CONFIG_TOKEN: ${NBGALLERY_CONFIG_TOKEN}
```
Comment (mcrutch) [inline docker-compose-with-jupyter.yml:10]: base-notebook does not support NBGALLERY_CONFIG_TOKEN or NBGALLERY_CONFIG_PASSWORD.  @jeffyjefflabs said he might ad support for it in the future (https://github.com/nbgallery/docker-images/issues/8#issuecomment-531998137) but it is not a current issue, so should probably just remove these from the docker-compose if we are using base-notebook.

Comment (jeffyjefflabs) [reply docker-compose-with-jupyter.yml:10]: Agreed

**Comment by jeffyjefflabs** [comment]
> @jeffyjefflabs Should we just make the Jupyter container run on 8888 since that is the default anyway and if they look at the logs to get the token, they will just be able to grab that URL rather than have to change the port to 80?

Yeah, good catch, I like that idea.

**Pull request merged**